design-documents/graph-ql/coverage/add-items-to-cart/AddSimpleProductToCart.graphqls

```diff
@@ -0,0 +1,78 @@
+type Mutation {
+    addSimpleProductsToCart(input: AddSimpleProductsToCartInput): AddSimpleProductsToCartOutput
+    updateSimpleProductsInCart(input: UpdateSimpleProductsInCartInput): UpdateSimpleProductsInCartOutput
+}
+
+input UpdateSimpleProductsInCartInput {
+    cart_id: String!
+    cartItems: [UpdateSimpleProductCartItemInput!]!
+}
+
+input UpdateSimpleProductCartItemInput {
+    details: UpdateCartItemDetailsInput!
```

Comment:
Why do we need one more level like `UpdateCartItemDetailsInput`

Now request is
```
cartItems: [
        {
          data: {
            sku: "product_dynamic_24494", 
            qty: 1
          }, 
          customizable_options: [
            {id: 5, value: "test"},
           ]
        }
      ]
```

Can we use next?
```
cartItems: [
        {
          sku: "product_dynamic_24494", 
          qty: 1,
          customizable_options: [
            {id: 5, value: "test"},
           ]
        }
      ]
``